test/components/DashboardTest.js

```diff
@@ -9,7 +9,7 @@ var DashboardConfigurationStore = require('../../src/stores/DashboardConfigurati
 describe('Dashboard', function () {
     it('loads the configuration from the store', function () {
         TestUtils.renderIntoDocument(<Dashboard />);
-        expect(DashboardConfigurationStore.get.mock.calls.length).toEqual(1);
+        expect(DashboardConfigurationStore.get.mock.calls.length).toEqual(0);
```

Comment:
Having a once over on the class it looks like the config is now being passed in through a prop rather than an ajax call so my guess is the assertion should be removed.